core-plugins/docs/File-batchsink.md

```diff
@@ -3,50 +3,28 @@
 
 Description
 -----------
-Writes to a filesystem in text, avro, or parquet format.
+Writes to a filesystem in various formats format.
 
-For the text format, each record is written out as delimited text.
-Non-string fields will be converted to strings using their ``toString()`` Java method,
-so fields should be limited to the string, long, int, double, float, and boolean types.
+For the csv, delimited, and tsv formats, each record is written out as delimited text.
+Complex types like arrays, maps, and records will be converted to strings using their
+``toString()`` Java method, so for practical use, fields should be limited to the
+string, long, int, double, float, and boolean types.
 
 All types are supported when using the avro or parquet format.
 
-
 Properties
 ----------
-**referenceName:** This will be used to uniquely identify this sink for lineage, annotating metadata, etc.
-
-**path:** The directory to write to. For example, ``hdfs://mycluster.net:8020/my/desired/location``.
+**Reference Name:** Name used to uniquely identify this sink for lineage, annotating metadata, etc.
 
-**suffix:** Time suffix to append to the path for each run of the pipeline. For example,
-``YYYY-MM-dd-HH-mm`` will take the start time of the pipeline run, convert it into
-year-month-day-hour-minute format, and append that to the path to get the final output directory.
-If not specified, no suffix is used.
+**Path:** Path to write to. For example, /path/to/output
 
-**delimiter:** Delimiter used to concatenate record fields. Defaults to a comma ','. (Macro-enabled)
-
-**jobProperties:** Advanced feature to specify any additional properties that should be used with the sink,
-specified as a JSON object of string to string. These properties are set on the job at runtime. (Macro-enabled)
+**Path Suffix:** Time format for the output directory that will be appended to the path.
+For example, the format 'yyyy-MM-dd-HH-mm' will result in a directory of the form '2015-01-01-20-42'.
+If not specified, nothing will be appended to the path."
 
 **Format:** Format to write the records in.
-The format must be one of 'json', 'avro', 'parquet', 'csv', 'tsv', or 'delimited'. (Macro-enabled)
+The format must be one of 'json', 'avro', 'parquet', 'csv', 'tsv', or 'delimited'.
 
 **Delimiter:** Delimiter to use if the format is 'delimited'.
-The delimiter will be ignored if the format is anything other than 'delimited'.
-
-**Schema:** Schema of the data to write. (Macro-enabled)
```

Comment:
Should we include some description about the schema? Since there can be some validation about the schema

Reply:
the sink doesn't actually validate schema, it just uses whatever is configured. The source does validation, which is mentioned in the format section.